alws/utils/beholder_client.py

```diff
@@ -23,6 +25,15 @@ async def get(self, endpoint: str,
         full_url = self._get_url(endpoint)
         async with aiohttp.ClientSession(headers=req_headers) as session:
             async with session.get(full_url, params=params) as response:
-                json = await response.json()
+                json_ = await response.json()
                 response.raise_for_status()
```

Comment:
You should call `raise_for_status()` before reading any data. And I'm not sure that aiohttp has the same method